setup.cfg

```diff
@@ -25,7 +25,7 @@ include_trailing_comma = True
 skip_glob = tests/testdata
 
 [flake8]
-extend-ignore = E203,E266,E501,C901,F401
+extend-ignore = E203,E266,E501,C901,F401,W503,B950,B901
```

Comment:
What are these three messages?

And why does this bump suddenly raise `B901`? That PR you mention was merged in 2016?

Reply:
I guess it's a breaking change from flake8 > 5 that change the default vs disabled message. The two other messages are line too long and line break before binary operator (I did not remember but it's in the commit message).